packages/taquito-rpc/src/types.ts

```diff
@@ -480,7 +480,7 @@ export interface OperationContentsSmartRollupOriginate {
   counter: string;
   gas_limit: string;
   storage_limit: string;
-  pvm_kind: PVMKind;
+  pvm_kind: string;
```

Comment:
Is `pvm_kind` that we want to create enum like the api doc? To prevent user from sending any string and got rejected by chain?